icechunk/src/conflicts/mod.rs

```diff
@@ -49,7 +49,7 @@ pub enum ConflictResolution {
 }
 
 #[async_trait]
-pub trait ConflictSolver {
+pub trait ConflictSolver: Send + Sync {
```

Comment:
Required for use in async python

Reply:
Hm this smells bad to me. Can we demand `Send + Sync` in the usage point instead?